### ISSUE TITLE
Projects using xunit.runner.console 2.3.0-beta2-build3683 cannot open

### ISSUE THREAD

**Issue by dfederm** [xunit/xunit#1273]
Package: xunit.runner.console
Version: 2.3.0-beta2-build3683

Opening a project is failing sure to invalid XML in xunit.runner.console.props (`<XunitConsoleNetCoreAppPath>` vs `</XunitConsolePath>`)
```
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

    <PropertyGroup>
        <XunitConsolePath>$(MSBuildThisFileDirectory)..\tools\net452\xunit.console.exe</XunitConsolePath>
        <XunitConsolePathX86>$(MSBuildThisFileDirectory)..\tools\net452\xunit.console.x86.exe</XunitConsolePathX86>
        <XunitConsoleNetCoreAppPath>$(MSBuildThisFileDirectory)..\tools\netcoreapp1.0\xunit.console.dll</XunitConsolePath>
    </PropertyGroup>

</Project>
```

**Comment by kzu** [comment]
Hitting this on the published nuget. Maybe make it unlisted and publish a new one?

**Comment by bradwilson** [comment]
Good point, I will un-publish it. I'll be pushing a beta 3 build sometime in the next week.

**Comment by masaeedu** [comment]
Is there a different feed with CI builds?

**Comment by bradwilson** [comment]
Grab a feed URL from the gallery: https://myget.org/gallery/xunit

**Comment by bradwilson** [comment]
We shipped 2.3 beta 3 today, which should resolve this for you without needing to use MyGet.
